reviews/views.py

```diff
@@ -354,14 +354,13 @@ def form_valid(self, form):
         proposal = form.instance.proposal
 
         if form.instance.continuation in [
-                Review.GO, Review.NO_GO, Review.GO_POST_HOC, Review.NO_GO_POST_HOC, Review.REVISION
+                Review.GO,
+                Review.NO_GO,
+                Review.GO_POST_HOC,
+                Review.NO_GO_POST_HOC,
+                Review.REVISION,
         ]:
-            proposal.status = Proposal.DECISION_MADE
-            proposal.status_review = form.instance.continuation in [
-                Review.GO, Review.GO_POST_HOC
-            ]
-            proposal.date_reviewed = timezone.now()
-            proposal.save()
+            proposal.mark_reviewed(form.instance.continuation)
```

Comment:
Github won't let me comment on the code below, this comment is about lines 364-372

The rest of this method looks so nice and clean now, but that part still follows the old doctrine. From what I can tell, it would be safe to consolidate that entire section in ``start_review_route`` (and _maybe_ also move that into a model) 

The same argument can be said for other views in this file, but that's getting a bit too far from the scope of this PR. It might be nice to create a seperate issue for this as well, it's a nice starter-issue for our new colleague when they start.

Reply:
I totally agree, but it's just out of scope for this PR. I don't feel like writing tests for all that right now.